dashboard/tasks.py

```diff
@@ -71,20 +67,12 @@ def release_lock():
     if acquire_lock():
         refresh_time_limit = datetime.utcnow() - timedelta(hours=MAX_HRS_MAIN_TASK)
```

Comment:
Can we make constants of the time values in `CachedEdxUserData.CACHE_EXPIRATION_DELTAS`, then make `MAX_HRS_MAIN_TASK` calculated based on the max value?